conf/20-look_and_feel

```diff
@@ -0,0 +1,3 @@
+# http://stackoverflow.com/questions/171563/whats-in-your-zshrc
```

Comment:
Also, look_and_feel is a wrong name. Call the file "nobeep" or something, as stupid as it seems to have a single line as a separate file, but the config-style of shellex is to have isolated units that can be ln'ed.